server/opendp_apps/dataverses/serializers.py

```diff
@@ -31,3 +31,9 @@ def save(self, **kwargs):
         self.validated_data['dv_general_token'] = dataverse_handoff.apiGeneralToken
         self.validated_data['dv_sensitive_token'] = dataverse_handoff.apiSensitiveDataReadToken
         return super().save()
+
```

Comment:
On line 25, for the DRF docs.
- switch: `fields = '__all__'` to 
   - `fields = ('dv_installation', 'user', 'dv_handoff')`